.github/workflows/test.yaml

```diff
@@ -92,7 +88,7 @@ jobs:
       fail-fast: false
       matrix:
         feature: [pm2]
-        engine: [binary] # ['node-api', 'binary']
+        clientEngine: [node-api] # ['node-api', 'binary']
```

Comment:
Why the switch / only testing node-api now?

Reply:
https://github.com/prisma/e2e-tests/pull/1954#discussion_r684442790

Reply:
Changed back to `['binary']`